sdk/core/azure-core-serializer-avro-apache/src/main/java/com/azure/core/serializer/avro/apache/ApacheAvroSerializer.java

```diff
@@ -17,11 +18,13 @@
 
 import java.io.InputStream;
 import java.io.OutputStream;
+import java.lang.reflect.Type;
 
 /**
  * Apache Avro based implementation of the {@link ObjectSerializer} interface.
  */
 public class ApacheAvroSerializer implements ObjectSerializer {
+    private final ClientLogger logger = new ClientLogger(ApacheAvroSerializer.class);
```

Comment:
Does this `ClientLogger` get used?

Reply:
Please ignore all changed in core serializer besides of PropertyName. I will get all your changes merged into my PR.